src/components/engagement/dto/update-engagement.dto.ts

```diff
@@ -2,7 +2,14 @@ import { Field, InputType, ObjectType } from '@nestjs/graphql';
 import { Type } from 'class-transformer';
 import { ValidateNested } from 'class-validator';
 import { stripIndent } from 'common-tags';
-import { CalendarDate, DateField, ID, IdField } from '../../../common';
+import { RichTextField } from '~/common/rich-text.scalar';
+import {
+  CalendarDate,
+  DateField,
+  ID,
+  IdField,
+  RichTextDocument,
+} from '../../../common';
```

Comment:
```suggestion
import {
  CalendarDate,
  DateField,
  ID,
  IdField,
  RichTextDocument,
  RichTextField,
} from '~/common';
```

Reply:
Yup, I'll get to that in the morning after the rename